test/unit/assorted/server_discovery_and_monitoring.spec.test.ts

```diff
@@ -268,6 +271,7 @@ function normalizeServerDescription(serverDescription) {
     // it as `Unknown`.
     serverDescription.type = 'Unknown';
   }
+  //serverDescription.error = serverDescription?.error?.message;
```

Comment:
this shouldn't be necessary, since you're already matching the `error.message` property correctly in the assertions 🙂